### ISSUE TITLE
[Beehave] Lookup table Github-link does not work

### ISSUE THREAD

**Issue by csckata** [BioDT/biodt-shiny#170]
Request to fix the existing Github-link located under the lookup-table and update the end-location to right place, as now it might be very unclear what the user should look out for.
